javascript/computer_science/space_complexity.md

```diff
@@ -59,7 +59,7 @@ function multiply(num1, num2) {
 }
 ```
 
-Here it should hopefully be clear that no matter the arguments we enter when we call the function, only two variables are created. It doesn't change. Therefore, we can consider the space this takes is always O(1).
+Here it should hopefully be clear that no matter the arguments we pass to the function call, only a single variable is created. It doesn't change. Therefore, we can consider the space this takes is always O(1).
```

Comment:
I don't feel great about the word "variable" here. The return value isn't being declared as a variable. I think a wording like this would be preferable:
```suggestion
Here it should hopefully be clear that no matter the arguments we pass to the function call, only a single value is created (the product of the numbers). It doesn't change. Therefore, we can consider the space this takes is always O(1).
```
What do you think?

Reply:
Since I am just going through this lesson, I didn't want to change the original conveyed message of 'variable', but yeah, I do feel as weird about it as you do. But yeah, functions shouldn't return variables since they were never declared in the first place. It's a simple operation, as you've described.

About the PR on Ruby, sure! That would be a pleasure. I'll do it in a jiffy.

I like your suggestion. Also, considering I'm not very acquainted with replying with new suggestions here on GitHub, I wanted to add to the suggestion you've made by changing the _"It doesn't change"_ part for _"This value doesn't change."_, for extra clarity. If you'd show me how to go about that, that would be lovely.